index.js

```diff
@@ -138,6 +138,7 @@ const featuresToIncludeTempUnprotectedExceptions = [
     'https',
     'navigatorCredentials',
     'navigatorInterface',
+    'newSettings',
```

Comment:
```suggestion
```

This isn't needed now (is only needed for features that support per domain exceptions anyway).